src/main/java/org/alfresco/repo/domain/schema/SchemaBootstrap.java

```diff
@@ -1050,7 +1101,42 @@ else if (!apply)
             executeScriptUrl(connection, scriptUrl);
         }
     }
-    
+
+    /**
+     * This method throws a LockFailedException in case the current Alfresco cluster node
+     * is NOT the node that is doing the schema bootstrap;
+     * This is the exception that signals that this node should wait for a while and retry
+     *
+     *
+     * Only one node from the cluster should do the initialization of the DB, so all other
+     * nodes should just retry to execute the schema checks after the initialization of
+     * the DB has ended;
+     *
+     */
+    private void ensureCurrentClusterMemberIsBootstrapping(Connection connection) throws Exception
+    {
+        if (isAnotherClusterMemberBootstrapping(connection))
+        {
+            logger.info("Another cluster node is doing a DB schema initialization or DB update. "

```

Comment:
"Another Alfresco cluster node is updating the DB.  Waiting for the other node to finish ..."